docs/3.0.0-beta.x/community/contribute-with-docs.md

```diff
@@ -0,0 +1,486 @@
+- Create a GitHub repo with your app for the **Tutorial** or for the code for the **How-to**.
+- Show this code to the Strapi Maintainer and address any feedback
+- Following the [Docs Style Guide](http://localhost:8080/documentation/3.0.0-beta.x/community/contribute-with-docs.html#docs-style-guide) below, write your Tutorial and make a pull request.
+
+#### Tips
+
+1. Make sure that following the steps results in a working app (for Tutorials) or working code (for How-tos).
+2. Smaller workable apps are better than large apps that have many moving parts.
+3. If you have any questions, please ask!
+
+### Adding / Modifying / Deleting Content from the Docs
+
+_Adding, modifying, and deleting content_ from the documentation follows the [guidelines for making a Pull Request](/3.0.0-beta.x/community/contribute-with-code.html#before-submitting-a-pull-request) to the Strapi monorepo. For correcting typos, or clarifications, this is a relatively straightforward process.
+
+**Tutorials** and **How-tos** are sometimes by a members of the community. A byline appears towards the title with a **Written by: Paul Bocuse**. If you make _significant_ revisions to the **Tutorial** or **How-tos**, then the byline would change and include your name, like so: **Original Author: Paul Bocuse, Revised by: Your Name**. You should link to your GitHub from your name.
+
+**Examples** are generally added to **Reference Topics** and **How-tos**. The examples are added directly in the existing content.
+
+### Submitting an Externally Hosted Tutorial
+
+If you have written a **Tutorial**, **How-to** or other teaching material that is _outside_ of the documentation. Then you may submit a pull request with a link to your guide on the [Articles](/3.0.0-beta.x/articles/) page.
+
+### Docs Style Guide
+
+This style guide exists to show and explain the standards and style of how our documentation is written. We are offering this style guide to help you stay consistent with the existing documentation.
+
+---
+
+#### Favor _Yarn_ commands over _npm_ commands
+
+We recommend that users develop using the Yarn package manager. Therefore, the documentation shows commands first using `yarn` and then showing the equivalent `npm` command. Please consider the below examples:
+
+**Example 1**
+
+- Install Strapi with the following commands:
+
+Using **Yarn**:
+
+```
+yarn create strapi-app my-project --quickstart
+```
+
+Using **npm/npx**:
+
+```
+npx create-strapi-app my-project --quickstart
+```
+
+**Example 2**
+
+- You can access the Strapi Help instructions from your command line with:
+
+Using **Yarn**:
+
+```
+yarn strapi
+```
+
+Using **npm**:
+
+```
+npm run strapi help
+```
+
+---
+
+#### When giving instructions, say, _"you"_ not _"we"_
+
+Clear and concise instructions and language can help individuals to get the full advantages from using the Docs. Whenever an action needs doing, you use the word, "you" rather than "we." To clarify this, consider the following examples:
```

Comment:
Maybe you should provide your source about why it is better to use 'you' than 'we' so you don't need the example below.

Reply:
For copywriters this is evident and I am not aware of any "official source" or "study". Most of our users, it is safe to say, are not copywriters. So I added this as an example to help us be able to write confident, clear and direct documentation.